### DISCUSSION TITLE
having trouble with Stable Audio node in Sound Lab

### DISCUSSION THREAD

**Discussion by phitch516** [Comfy-Org/ComfyUI#5182] [Q&A]
I installed Sound Lab from manager and Music Gen node works, but I get the Error no module named 'vector_quantize_pytorch' when using stable audio, downloaded vector quantize pytorch, same problem, looked at the scripts in pycharm and the same import statements that work for calling other modules looks the same as the one calling vector quantize pytorch. i am completely new to python and have only very basic coding skills, not sure what is going on. Here is snippet of traceback.
![Screenshot 2024-10-07 093910](https://github.com/user-attachments/assets/1156ea02-4a28-4485-911b-e93342188b4c)
Just started using comfyui a week or 2 ago, and it has been pretty intuitive using the nodes, and thank you in advance for any insight.

**Comment by jinja48** [comment]
I am having the same issue, was there ever a solution? Thanks in advance for any feedback.

**Comment by ltdrdata** [comment]
https://github.com/shadowcz007/comfyui-sound-lab/issues/8
